### PR TITLE
DOC-4655 - Release notes update

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
First tranche of release note updates for 4.11 Ent and Community

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
 
 ## New features and enhancements
 
+* This release includes improvements in the performance of deserializing AMQP data, which may result in performance improvements for LedgerGraph, Archiving and other CorDapps.
 
+* A new property, `previousPageAnchor`, has been added to `Vault.Page`. It is used to detect if the vault has changed while pages of a vault query have been loaded. If such a scenario is important to detect, then the property can be used to restart querying.
+
+  A example of how to use this property can be found in [Vault queries]({{< relref "cordapps/api-vault-query.md#query-for-all-states-using-a-pagination-specification-and-iterate-using-the-totalstatesavailable-field-until-no-further-pages-available-1" >}}).
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:34]: Same comment about "Vault queries" as the previous one.

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
+
+* When a notary worker is shut down, message ID cleanup is now performed as the last shutdown activity, rather than the first; this prevents a situation where the notary worker might still appear to be part of the notary cluster and receiving client traffic while shutting down.
+
+* Previously, when configured to use confidential identities and the Securosys PrimusX HSM, it was possible for Corda to fail to generate a wrapped key-pair for a new confidential identity. This would cause a temporary key-pair to be leaked, consuming resource in the HSM. This issue occurred when:
+
+  * the Securosys HSM was configured in a master-clone cluster
+
+  * the master HSM had failed and Corda had failed-over to use the clone HSM
+
+  * there was an attempt to create a transaction using confidential identities
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:57]:
```suggestion
  * There was an attempt to create a transaction using confidential identities
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
+  [INFO ] 2023-02-03T17:00:12,911Z [main] statemachine.MultiThreadedStateMachineManager. - Started loading finished flows {}
+  [INFO ] 2023-02-03T17:00:28,437Z [main] statemachine.MultiThreadedStateMachineManager. - Loaded 9001 finished flows {}
+  [INFO ] 2023-02-03T17:00:43,606Z [main] statemachine.MultiThreadedStateMachineManager. - Loaded 24001 finished flows {}
+  [INFO ] 2023-02-03T17:00:46,650Z [main] statemachine.MultiThreadedStateMachineManager. - Number of finished flows : 27485 {}
+  ```
+  
+* Previously, if a node was configured to use two different slots on the Luna HSM (for example using one slot for node identities and a separate slot for the confidential identities), this failed. This issue has now been resolved. 
+
+  {{< warning >}}
+  However as a result of this fix, you need to make sure the Luna client your are using is version 10.4.0 or later.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:93]:
```suggestion
  However, as a result of this fix, you need to make sure the Luna client your are using is version 10.4.0 or later.
```

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
 
+* Updated documentation for both `.startNodes()` and `.stopNodes()` of `MockNetwork` to indicate that restarting nodes is not supported.
+
+* A fix for cache eviction has been applied where an issue resulted in an incorrect contract verification status while a database transaction was in progress during contract verification.
+
+* When a notary worker is shut down, message ID cleanup is now performed as the last shutdown activity, rather than the first; this prevents a situation where the notary worker might still appear to be part of the notary cluster and receiving client traffic while shutting down.
+
+* Previously, when configured to use confidential identities and the Securosys PrimusX HSM, it was possible for Corda to fail to generate a wrapped key-pair for a new confidential identity. This would cause a temporary key-pair to be leaked, consuming resource in the HSM. This issue occurred when:
+
+  * the Securosys HSM was configured in a master-clone cluster
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:53]: Done.

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 5]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
+
+  * the master HSM had failed and Corda had failed-over to use the clone HSM
+
+  * there was an attempt to create a transaction using confidential identities
+
+  The issue is now resolved. When generating a wrapped key-pair the temporary key-pair is not persisted in the HSM and thus cannot be leaked.
+
+  On applying this update the PrimusX JCE should be upgraded to version 2.3.4 or later.
+
+  There is no need to upgrade the HSM firmware version for this update but it is recommended to keep the firmware up to date as a matter of course. Currently the latest firmware version if 2.8.50.
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:63]: Done

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 6]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
+  However as a result of this fix, you need to make sure the Luna client your are using is version 10.4.0 or later.
+  {{</ warning >}}
+  
+* The default value for the node configuration value `cryptoServiceTimeout` has been increased from 1 second to 10 seconds.
+
+* Flow checkpoint dumps now include a `status` field which shows the status of the flow; in particular, whether it is hospitalized or not.
+
+* Debug logging of the Artemis server has been added.
+
+* When a notary worker is shut down, message ID cleanup is now performed as the last shutdown activity, rather than the first; this prevents a situation where the notary worker might still appear to be part of the notary cluster and receiving client traffic while shutting down.
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:102]: Done

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:93]: Done

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 7]
```diff
@@ -27,23 +27,90 @@ For more information about platform versions, see [Versioning](cordapps/versioni
...
+
+* A fix for cache eviction has been applied where an issue resulted in an incorrect contract verification status while a database transaction was in progress during contract verification.
+
+* When a notary worker is shut down, message ID cleanup is now performed as the last shutdown activity, rather than the first; this prevents a situation where the notary worker might still appear to be part of the notary cluster and receiving client traffic while shutting down.
+
+* Previously, when configured to use confidential identities and the Securosys PrimusX HSM, it was possible for Corda to fail to generate a wrapped key-pair for a new confidential identity. This would cause a temporary key-pair to be leaked, consuming resource in the HSM. This issue occurred when:
+
+  * the Securosys HSM was configured in a master-clone cluster
+
+  * the master HSM had failed and Corda had failed-over to use the clone HSM
```
[the remark this answers is not in the document]
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:55]: Done

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:57]: Done

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**